packages/kit/src/cli.js

```diff
@@ -200,15 +200,34 @@ prog
 		try {
 			if (H) throw new Error('-H is no longer supported — use --https instead');
 
-			await check_port(port);
-
 			process.env.NODE_ENV = process.env.NODE_ENV || 'production';
 
-			const { preview } = await import('./core/preview/index.js');
+			const { sveltekit_plugin } = await import('./core/preview/index.js');
+			const svelte_config = await load_config();
+			const vite_config = await svelte_config.kit.vite();
+
+			/** @type {import('vite').UserConfig} */
+			const config = {
+				...vite_config,
+				plugins: [...(vite_config.plugins || []), sveltekit_plugin]
```

Comment:
```suggestion
				plugins: [...(vite_config.plugins || []), ...sveltekit_plugin]
```

Reply:
`sveltekit_plugin` is just a single plugin, so it wouldn't make sense to spread it. This is different than dev because in dev we need `vite-plugin-svelte` which adds a second plugin, but in preview the `.svelte` files have already been compiled, so there's no need for `vite-plugin-svelte`